db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_remove_vm_finished.yaml

```diff
@@ -9,8 +9,8 @@ object:
     description: 
   fields:
   - rel4:
-      value: "/System/event_handlers/src_vm_disconnect_storage"
-  - rel5:
       value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_unregister&param="
+  - rel5:
+      value: "/System/event_handlers/event_action_refresh_sync?target=src_vm"
```

Comment:
I assume since he's using the result of the refresh in `disconnect_storage` he needs to wait for the refresh to finish before running the next step here https://github.com/ManageIQ/manageiq/pull/12079/files#diff-91a604e15a08c604ed26c6db2b67533dR16